<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralMainWnd</class>
 <widget class="QMainWindow" name="GeneralMainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneralMainWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>右下角的提示框CornerPromptBox 的使用</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>时间 秒</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxSecond">
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxUsingAnimation">
         <property name="text">
          <string>用动画</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonCornerPromptBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonShowText">
         <property name="text">
          <string>显示文字</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonShowWidget">
         <property name="text">
          <string>显示widget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>自己封装的Diglog的使用</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonHengPai">
         <property name="text">
          <string>水平布局</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="labeldialogResult">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonChuizhi">
         <property name="text">
          <string>垂直布局</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonShowDialog">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>HelpQComboBox 相关函数</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxEmpty"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonGetComboBoxAllItemtext">
         <property name="toolTip">
          <string>qhGetComboBoxAllItemtext</string>
         </property>
         <property name="text">
          <string>qhGetComboBoxAllItemtext</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxDisableEdit">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>abc</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditInHelpQComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBoxEnableEdit">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>abc</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonSetComboBoxValue">
         <property name="toolTip">
          <string>qhSetComboBoxValue</string>
         </property>
         <property name="text">
          <string>qhSetComboBoxValue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>InitNote</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonInitNote">
           <property name="text">
            <string>InitNote</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonInitNoteRename">
           <property name="text">
            <string>InitNoteRename</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Dump </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonDumpEventFilters">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>qhDumpEventFilters</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonDumpObjectTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>qhDumpObjectTree</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>得到多个文件对话框</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBoxFileMode">
         <item>
          <property name="text">
           <string>AnyFile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ExistingFile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Directory</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ExistingFiles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DirectoryOnly </string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxRepeatFileMode">
         <item>
          <property name="text">
           <string>ERepeatFilesNone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ERepeatFilesWarn</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ERepeatFilesWarnForbit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ERepeatFilesForbit</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonShowDlg">
         <property name="text">
          <string>显示对话框</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>338</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
